fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogFragment.kt

```diff
@@ -40,15 +72,174 @@ class TranslationsDialogFragment : BottomSheetDialogFragment() {
+                }
+
+                var translationsHeightDp by remember {
+                    mutableStateOf(0.dp)
+                }
+
+                var translationsOptionsHeightDp by remember {
+                    mutableStateOf(0.dp)
+                }
+
+                var translationsWidthDp by remember {
+                    mutableStateOf(0.dp)
+                }
+
+                val density = LocalDensity.current
+
+                TranslationDialogBottomSheet {
+                    TranslationsAnimation(
+                        translationsVisibility = translationsVisibility,
+                        density = density,
+                        translationsOptionsHeightDp = translationsOptionsHeightDp,
+                    ) {
+                        if (translationsVisibility) {
+                            Column(
+                                modifier = Modifier.onGloballyPositioned { coordinates ->
+                                    translationsHeightDp = with(density) {
+                                        coordinates.size.height.toDp()
+                                    }
+                                    translationsWidthDp = with(density) {
+                                        coordinates.size.width.toDp()
+                                    }
+                                },
+                            ) {
+                                ShowTranslations(
+                                    onDismiss = {
+                                        translationsVisibility = false
+                                    },
+                                )
+                            }
+                        }
+                    }
+
+                    TranslationsOptionsAnimation(
+                        translationsVisibility = !translationsVisibility,
+                        density = density,
+                        translationsHeightDp = translationsHeightDp,
+                        translationsWidthDp = translationsWidthDp,
+                    ) {
+                        if (!translationsVisibility) {
+                            Column(
+                                modifier = Modifier.onGloballyPositioned { coordinates ->
+                                    translationsOptionsHeightDp = with(density) {
+                                        coordinates.size.height.toDp()
+                                    }
+                                },
+                            ) {
+                                ShowTranslationOptions(
+                                    onDismiss = {
+                                        translationsVisibility = true
+                                    },
+                                )
+                            }
+                        }
+                    }
+                }
             }
         }
     }
+
+    @Composable
+    private fun TranslationDialogBottomSheet(content: @Composable () -> Unit) {
```

Comment:
```suggestion
    private fun TranslationsDialogBottomSheet(content: @Composable () -> Unit) {
```
Note we are using `Translations` as our prefix everywhere already.

Reply:
Done